### PR TITLE
Update Base Function structure

### PR DESCRIPTION
**Pull request by elliotstoner**

### CODE REVIEW

File: services/azure-pipelines.yml [hunk 1]
```diff
@@ -1,106 +1,49 @@
-################################################################################
```
Comment (tarrantrj) [inline services/azure-pipelines.yml]: This change should have been left out and not included in this PR.

Comment (tarrantrj) [reply services/azure-pipelines.yml]: I updated it

Comment (elliotstoner) [reply services/azure-pipelines.yml]: Weird - my file matches what's on captech's repo?
https://github.com/elliotstoner/wearable-smart-elevator/blob/master/services/azure-pipelines.yml

Comment (elliotstoner) [reply services/azure-pipelines.yml]: Oh I didn't realize you had already updated it lol. It's also weird that it was wrong in the first place, on the merge conflict I told it to take your changes. Oh well 🤷🏼‍♂️

Comment (tarrantrj) [reply services/azure-pipelines.yml]: Meh PR caught it 👍

**Pull request merged**